### PR TITLE
Con 768 EOI remove links

### PR DESCRIPTION
**Pull request by Stephek**

### CODE REVIEW

File: src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerTeamController.cs [hunk 1]
```diff
@@ -50,7 +50,7 @@ public class EmployerTeamController : BaseController
         public async Task<ActionResult> Index(string hashedAccountId, string reservationId)
         {
             var response = await GetAccountInformation(hashedAccountId);
-            if (FeatureToggles.Features.HomePage.Enabled || !HasPayeScheme(response.Data) && !HasOrganisation(response.Data))
+            /*if (FeatureToggles.Features.HomePage.Enabled || !HasPayeScheme(response.Data) && !HasOrganisation(response.Data))
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerTeamController.cs]: don't think this was intentionally committed

File: src/SFA.DAS.EmployerAccounts.Web/Extensions/AccountDashboardViewModelExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                return false;
+            }
+
+            return true;
+        }
+
+        public static bool ShowYourFundingReservationsLink(this AccountDashboardViewModel model)
+        {
+            if(
+                model.ApprenticeshipEmployerType == ApprenticeshipEmployerType.NonLevy ||
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerAccounts.Web/Extensions/AccountDashboardViewModelExtensions.cs]: I think this should be AND logic. EmployerType.NonLevy && AgreementType.EOI

File: src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerTeamController.cs (hunk 1 above)
Comment (lazcool) [reply src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerTeamController.cs]: Code uncommented.

File: src/SFA.DAS.EmployerAccounts.Web/Extensions/AccountDashboardViewModelExtensions.cs (hunk 2 above)
Comment (lazcool) [reply src/SFA.DAS.EmployerAccounts.Web/Extensions/AccountDashboardViewModelExtensions.cs]: Perhaps the intent was (in effect) `(NonLevy || NonLevy) && EOI`?

Comment (Stephek) [reply src/SFA.DAS.EmployerAccounts.Web/Extensions/AccountDashboardViewModelExtensions.cs]: There are NonLevy/Levy states on the Employer and the Agreement.  The two are mutually exclusive according to the requirements.  So it is show link if Employer is non levy OR show if Agreement is non levy AND EOI.

Comment (cofaulco) [reply src/SFA.DAS.EmployerAccounts.Web/Extensions/AccountDashboardViewModelExtensions.cs]: Ok I think this is fine actually as Non-levy users who have signed a Levy agreement should be able to see both links for finance

Comment (Stephek) [reply src/SFA.DAS.EmployerAccounts.Web/Extensions/AccountDashboardViewModelExtensions.cs]: Updated

**Pull request merged**